Vagrantfile

```diff
@@ -0,0 +1,15 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+require 'fileutils'
+
+riot_vagrantfile = File.expand_path('RIOT/Vagrantfile')
+RIOTBASE ||= "RIOT/"
+load riot_vagrantfile if File.exists?(riot_vagrantfile)
```

Comment:
if the `RIOT` submodule (path) is not available or checkout failed and `RIOT/Vagrantfile` is missing this fails with:
```
Bringing machine 'default' up with 'virtualbox' provider...
There are errors in the configuration of this machine. Please fix
the following errors and try again:

vm:
* A box must be specified.
```

IMHO it would be nice to have a more _speaking_ error message suggesting that the issue might be related a failure with the RIOT submodule. So add something like an _else_ path and exit after error message without further processing.